package.json

```diff
@@ -9,7 +9,9 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "^1.91.0",
+    "@aws-cdk/assert": "2.12.0",
```

Comment:
ah just saw this. I think you need only `assert` as a `devDependency`. cdk-lib and construct are only needed in `peerDependencies` (where assert is not needed :D). But not 100% sure, so maybe give it a quick try.

Reply:
You're right! It does work. Good spot 👌